db/schema.rb

```diff
@@ -13,6 +13,7 @@
 ActiveRecord::Schema.define(version: 2020_10_24_145705) do
 
   # These are extensions that must be enabled in order to support this database
+  enable_extension "pg_trgm"
```

Comment:
This is not needed anymore

Reply:
Yes we do `word_similarity` is a trgm function